client/modules/User/pages/ResetPasswordView.jsx

```diff
@@ -83,7 +84,11 @@ function mapDispatchToProps(dispatch) {
 function validate(formProps) {
   const errors = {};
   if (!formProps.email) {
-    errors.email = 'Please enter an email';
+    errors.email = 'Please enter an email.';
+  } else if (
+    // eslint-disable-next-line max-len
+    !formProps.email.match(/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/i)) {
```

Comment:
there's a file called `client/utils/reduxFormUtils.js` that has validation functions, including a regex to validate an email address. i would put a function in there and use that regex!

Reply:
@catarak Okay, I will use that for file functions for validations.